lib/mayaUsd/resources/ae/usdschemabase/ae_template.py

```diff
@@ -233,12 +233,26 @@ def __init__(self, ufeSceneItem):
         if cmds.optionVar(exists='attrEditorIsLongName'):
             self.useNiceName = (cmds.optionVar(q='attrEditorIsLongName') ==1)
 
+        self.addedMaterialSection = False
+
+        self.suppressArrayAttribute()
+
+        # Build the list of schemas with their associated attributes.
+        schemasAttributes = {
+            'extraAttributes' : [],
+            'metadata' : [],
+        }
+        
+        schemasAttributes.update(self.findAppliedSchemas())
+        schemasAttributes.update(self.findClassSchemas())
+        schemasAttributes.update(self.findSpecialSections())
+
```

Comment:
The create function are replace with find ones. I accumulate the map of schema type name -> corresponding attributes.